src/components/Users/UserAvatar.tsx

```diff
@@ -74,8 +77,9 @@ export default function UserAvatar({
       pathParams: { username },
     });
     if (res?.ok) {
-      toast.success(t("profile_picture_deleted"));
       refetchUserData?.();
+      await queryClient.invalidateQueries({ queryKey: ["currentUser"] });
```

Comment:
we can skip awaiting the query invalidation. toast can be shown right after the mutation has upload/delete has completed without waiting for invalidation to complete.
```suggestion
      queryClient.invalidateQueries({ queryKey: ["currentUser"] });
```